server/src/main/java/org/elasticsearch/action/admin/cluster/bootstrap/GetDiscoveredNodesAction.java

```diff
@@ -23,7 +23,7 @@
 
 public class GetDiscoveredNodesAction extends Action<GetDiscoveredNodesResponse> {
     public static final GetDiscoveredNodesAction INSTANCE = new GetDiscoveredNodesAction();
-    public static final String NAME = "cluster:monitor/discovered_nodes";
+    public static final String NAME = "internal:cluster/discovered_nodes";
```

Comment:
I don't immediately see how this would help, since both are covered by system privileges:

https://github.com/elastic/elasticsearch/blob/9f86e996fecd8a6e6aa4a2b3c5e2580e1c3171db/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/SystemPrivilege.java#L19-L28

Not that it _won't_ help, just that I don't understand the mechanism.